src/main/java/com/example/WatchNext/model/Movies.java

```diff
@@ -0,0 +1,134 @@
+package com.example.WatchNext.model;
+
+import com.sun.istack.NotNull;
+import org.hibernate.validator.constraints.URL;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+@Entity
+public class Movies {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String title;
```

Comment:
It would be nice to have consistency and add @Collumn annotation  to see the name of the column from database...my idea is that if I am going to read the entity model I should be able to know how the database model will look without any look over SQL.